src/Animal.java

```diff
@@ -0,0 +1,22 @@
+import java.util.Random;
+
+public class Animal {
+
+    private final String name;
+
+    public Animal(String name) {
+        this.name = name;
```

Comment:
Where is an `animalCounter`? Can't find it in this class. It's supposed to be here.